public_html/user_id.php

```diff
@@ -2,6 +2,7 @@
 
 require_once 'common.php';
 
+// TODO: Is it useful to provide a batch query?
```

Comment:
We don't actually use this endpoint at all anymore, as we query  mediawiki directly to get this info. It's used only when adding a new user, which we don't do in batches; it's either immediately after a user signs in via OAuth for the first time, or immediately when a user is added manually to a course by the instructor.